mrbgems/mruby-sprintf/src/sprintf.c

```diff
@@ -194,7 +194,7 @@ check_name_arg(mrb_state *mrb, int posarg, const char *name, int len)
 
 #define GETNAMEARG(id, name, len) (\
   check_name_arg(mrb, posarg, name, len),\
-  (posarg = -2, mrb_hash_fetch(mrb, get_hash(mrb, &hash, argc, argv), id, mrb_undef_value())))
+  (posarg = -2, mrb_hash_fetch(mrb, get_hash(mrb, &hash, (int)argc, argv), id, mrb_undef_value())))
```

Comment:
How about making `argc` of `get_hash` to `mrb_int`?

Reply:
Fixed.